ibis-server/tests/routers/v2/connector/test_trino.py

```diff
@@ -402,6 +402,15 @@ def test_metadata_list_constraints(trino: TrinoContainer):
         result = response.json()
         assert len(result) == 0
 
+    def test_metadata_db_version(trino: TrinoContainer):
+        connection_info = _to_connection_info(trino)
+        response = client.post(
+            url=f"{base_url}/metadata/version",
+            json={"connectionInfo": connection_info},
+        )
+        assert response.status_code == 200
+        assert response.text is not None
```

Comment:
Same as the issue for clickhouse.

Reply:
We use the `latest` tag. It is not a static version.